examples/secure-tracing/README.md

```diff
@@ -0,0 +1,112 @@
+# Build A Secure Trace Collection Pipeline 
```

Comment:
Having examples is great, but it's also quite difficult for us to maintain them over time. To make sure we can keep this working, it would be necessary to add a test that runs the example on CI and checks things are working. That way we can avoid accidentally breaking the setup

Reply:
@mx-psi, do you have suggestions on how to do this ? I updated Makefile and added this as part of `build-examples`.

Reply:
This makes sense to me, although not sure why the `build-examples:` line on the Makefile was removed?